tools/wake-format/formatter.h

```diff
@@ -347,6 +347,85 @@ struct FmtPredicate<std::initializer_list<T>> {
   bool operator()(wcl::doc_builder& builder, ctx_t ctx, CSTElement& node) { return set[node.id()]; }
 };
 
+struct EpsilonCase {
```

Comment:
Is this different from an epsilon formatter?

Reply:
yeah, Epsilon doesn't have `return false`